app/src/main/java/turtler/voyageur/fragments/ViewPagerContainerFragment.java

```diff
@@ -99,30 +99,35 @@ private void populateTripUI() {
             public void done(Trip object, ParseException e) {
                 trip = object;
                 collapsingToolbarLayout.setTitle(trip.getName().toString());
-                trip.getTripCreatorRelation().getQuery().findInBackground(new FindCallback<User>() {
-                    @Override
-                    public void done(List<User> users, ParseException e) {
-                        if (users != null) {
-                            user = users.get(0);
-                            View b = parentActivity.findViewById(R.id.fabAddEvent);
-                            if (user.getEmail().equals(ParseUser.getCurrentUser().getEmail())) {
-                                b.setVisibility(View.VISIBLE);
-                            }
-                            else {
-                                b.setVisibility(View.GONE);
+
+                    trip.getTripCreatorRelation().getQuery().findInBackground(new FindCallback<User>() {
```

Comment:
instead of `trip.getTripCreatorRelation().getQuery().findInBackground(new FindCallback<User>() {` can we just do `trip.getTripFriendsRelation().getQuery().findInBackground(new FindCallback<User>() {`